pkg/networkservice/chains/forwarder/server.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package forwarder provides utility method for building interpose endpoint
+package forwarder
```

Comment:
Should we rename it to xconnect?

Reply:
I am not quite sure about the naming here. On the one hand we have a special name - `interpose endpoint` for this, but on the other hand almost everywhere in the code, in the samples, in tests we name it as `forwarder`.
So I suppose that `forwarder` here is a better name for the package here than `xconnect` or `interpose`.